cranelift-codegen/meta/src/lib.rs

```diff
@@ -40,8 +40,8 @@ pub fn generate(isas: &Vec<isa::Isa>, out_dir: &str) -> Result<(), error::Error>
     gen_inst::generate(
         all_inst_groups,
         &shared_defs.format_registry,
-        "new_opcodes.rs",
-        "new_inst_builder.rs",
+        "opcodes.rs",
+        "inst_builder.rs",
```

Comment:
Could you please remove generation of these files from meta-python at https://github.com/CraneStation/cranelift/blob/e9a81f6cce11c8115da3116bbfb205fb4e04f962/cranelift-codegen/meta-python/gen_instr.py#L794-L812

Reply:
I can remove most of gen_instr, but some functions it contains are still in use by the legalizer; I'll move them there.